src/EditorFeatures/Test/EditorServicesTest.csproj

```diff
@@ -361,4 +362,4 @@
     <Import Project="..\..\..\build\Roslyn.Toolsets.Xunit.targets" />
     <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Just had a look and it does appear that VS removes the trailing newline. Tried reverting and including the new test file and it did it again.